operations/helm/charts/mimir-distributed/templates/memcached/_memcached-statefulset.tpl

```diff
@@ -0,0 +1,97 @@
+{{/*
+memcached StatefulSet
+*/}}
+{{- define "mimir.memcached.statefulSet" -}}
+{{ with (index $.ctx.Values $.component) }}
+{{- if .enabled -}}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "mimir.resourceName" (dict "ctx" $.ctx "component" $.component) }}
+  labels:
+    {{- include "mimir.labels" (dict "ctx" $.ctx "component" "memcached") | nindent 4 }}
+  annotations:
+    {{- toYaml .annotations | nindent 4 }}
+spec:
+  podManagementPolicy: {{ .podManagementPolicy }}
+  replicas: {{ .replicas }}
+  selector:
+    matchLabels:
+      {{- include "mimir.selectorLabels" (dict "ctx" $.ctx "component" $.component) | nindent 6 }}
+  updateStrategy:
+    {{- toYaml .statefulStrategy | nindent 4 }}
+  serviceName: {{ template "mimir.fullname" $.ctx }}-{{ $.component }}
+
+  template:
```

Comment:
please add support for global.extraEnv, global.extraEnvFrom and global.podAnnotations when rebasing (see #2099 #2031 )